### PR TITLE
Fix for https://github.com/icesat2py/icepyx/issues/247

### PR DESCRIPTION
**Pull request by learn2phoenix**
- Copies the functionality from `get_pypi_stats.ipynb` to `get_pypi_stats.py`
- adds a workflow `get_pypi_stats.yml`

### CODE REVIEW

File: .github/workflows/get_pypi_stats.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+name: Get PyPi Stats
+on:
+  schedule:
+    # runs once a week on sunday
```
Comment (JessicaS11) [inline .github/workflows/get_pypi_stats.yml:4]:
```suggestion
    # runs once a month on the first
```

**Comment by JessicaS11** [comment]
@learn2phoenix Thanks - this looks great! I updated a comment in the yaml, and I'd say we can go ahead and delete the original ipynb.

**Comment by learn2phoenix** [comment]
@JessicaS11 Both things done.

**Comment by JessicaS11** [comment]
@learn2phoenix Thanks for contributing! Please add yourself to our list of contributors!

**Pull request merged**